general/general-contributing.asciidoc

```diff
@@ -1,15 +1,11 @@
 = Contributing Guide
 
-include::Contributing-Code.asciidoc[leveloffset=1]
+include::../.github/CONTRIBUTING.asciidoc[leveloffset=1]
 
-include::Contributing-Code-of-Conduct.asciidoc[leveloffset=1]
-
-include::Contributing-Development-Guidelines.asciidoc[leveloffset=1]
+include::../.github/CODE_OF_CONDUCT.md[leveloffset=1]
```

Comment:
This wont work because asciidoctor includes .asciidoc and .adoc files but not .md. I suggest you to create an .asciidoc file with the same content and include it (if you want to keep this .md, otherwise just change its extension)